tools/tune/src/src_export_table_2s.m

```diff
@@ -82,45 +82,44 @@
         um_tmp = unique_modes(i,:);
         if isequal(um_tmp(1:2),[1 1]) || isequal(um_tmp(1:2),[0 0])
         else
-                fprintf(fh, '#include <%ssrc_%s_%d_%d_%d_%d.h>\n', ...
-                        ppath, prof_ctype, um_tmp(1:4));
+                fprintf(fh, '#include \"src_%s_%d_%d_%d_%d.h\"\n', ...
+                        prof_ctype, um_tmp(1:4));
 
                 n = all_modes(ia(i), 5);
                 a = all_modes(ia(i), 6);
                 b = all_modes(ia(i), 7);
                 sfl = sfl +taps_2s(n, a, b); % Count sum of filter lengths
         end
 end
-fprintf(fh, '#include <sof/audio/src/src.h>\n');
 fprintf(fh, '#include <stdint.h>\n');
 fprintf(fh,'\n');
 
 fprintf(fh, '/* SRC table */\n');
 switch ctype
         case 'float'
-                fprintf(fh, '%s fir_one = 1.0;\n', vtype);
+                fprintf(fh, 'const %s fir_one = 1.0;\n', vtype);
                 fprintf(fh, 'struct src_stage src_double_1_1_0_0 =  { 0, 0, 1, 1, 1, 1, 1, 0, 1.0, &fir_one };\n');
                 fprintf(fh, 'struct src_stage src_double_0_0_0_0 =  { 0, 0, 0, 0, 0, 0, 0, 0, 0.0, &fir_one };\n');
```

Comment:
what about these? Aren't they `const` now too?

Reply:
Yep, you are right!